### ISSUE TITLE
Can't install on Mint 21.04 [Easy workaround]

### ISSUE THREAD

**Issue by MarkJeronimus** [edouard-lopez/byzanz-gui#5]
I get these errors. Note that this is all unattended, including the final Abort which looks as if it asked my user input but didn't.

```
$ curl --location https://git.io/record-gif.sh | bash -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   661  100   661    0     0   1872      0 --:--:-- --:--:-- --:--:--  1872
Installing byzanz
Reading package lists... Done
Building dependency tree       
Reading state information... Done
autoconf is already the newest version (2.69-11.1).
byzanz is already the newest version (0.3.0+git20160312-3).
0 upgraded, 0 newly installed, 0 to remove and 5 not upgraded.
Installing lolilolicon/xrectsel and dependencies
Cloning into '/tmp/xrectsel'...
remote: Enumerating objects: 10, done.
remote: Counting objects: 100% (10/10), done.
remote: Compressing objects: 100% (8/8), done.
remote: Total 10 (delta 0), reused 5 (delta 0), pack-reused 0
Unpacking objects: 100% (10/10), 15.70 KiB | 1.12 MiB/s, done.
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal 
autoreconf: configure.ac: tracing
autoreconf: configure.ac: creating directory build-aux
autoreconf: configure.ac: not using Libtool
autoreconf: running: /usr/bin/autoconf
autoreconf: running: /usr/bin/autoheader
autoreconf: running: automake --add-missing --copy --no-force
configure.ac:4: installing 'build-aux/compile'
configure.ac:3: installing 'build-aux/install-sh'
configure.ac:3: installing 'build-aux/missing'
Makefile.am: installing 'build-aux/depcomp'
autoreconf: Leaving directory `.'
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for gcc... gcc
checking whether the C compiler works... no
configure: error: in `/tmp/xrectsel':
configure: error: C compiler cannot create executables
See `config.log' for more details
Reading package lists... Done
Building dependency tree       
Reading state information... Done
automake is already the newest version (1:1.16.1-4ubuntu6).
automake set to manually installed.
The following additional packages will be installed:
  libc-dev-bin libc6-dev libcrypt-dev libltdl-dev libpthread-stubs0-dev libxau-dev libxcb1-dev libxdmcp-dev x11proto-core-dev x11proto-dev xorg-sgml-doctools xtrans-dev
Suggested packages:
  glibc-doc libtool-doc gfortran | fortran95-compiler gcj-jdk libx11-doc libxcb-doc
The following NEW packages will be installed:
  libc-dev-bin libc6-dev libcrypt-dev libltdl-dev libpthread-stubs0-dev libtool libx11-dev libxau-dev libxcb1-dev libxdmcp-dev x11proto-core-dev x11proto-dev
  xorg-sgml-doctools xtrans-dev
0 upgraded, 14 newly installed, 0 to remove and 5 not upgraded.
Need to get 4,463 kB of archives.
After this operation, 28.0 MB of additional disk space will be used.
Do you want to continue? [Y/n] Abort.
```

**Comment by MarkJeronimus** [comment]
`checking whether the C compiler works... no` fixed itself after I manually executed `sudo apt-get install libx11-dev automake libtool` (a line that comes *after* in the sh file). Everything works with this fix.

**Comment by edouard-lopez** [comment]
What's inside the `See `config.log' for more details` ? Maybe we should move the `apt install` command a bit earlier

**Comment by MarkJeronimus** [comment]
I can't find said log file. Where would I find it?

**Comment by edouard-lopez** [comment]
It's related to 

```
configure: error: in `/tmp/xrectsel':
configure: error: C compiler cannot create executables
See `config.log' for more details
```
So, probably in `/tmp/config.log` or `/tmp/xrectsel/config.log`. But I reckon moving the [installation of `libx11-dev automake libtool`](https://github.com/edouard-lopez/record-gif.sh/blob/master/install.sh#L11 )before cloning `xrectsel` would be enough. 

Would you like to submit an MR?

As for not asking your password, it's probably because you used `sudo` earlier and the "password session" was still available.

**Comment by MarkJeronimus** [comment]
it's `/tmp/xrectsel/config.log`, but it's overwritten with the successful run.

What password prompt?? It asked for Y/n

Why would I make a PR/MR for something so small? (also referring to issue 6)

**Comment by edouard-lopez** [comment]
Wasn't clear about the abort. You find and documented a workaround that's fine with me

**Comment by MarkJeronimus** [comment]
I'm running into the same issue again, on same Mint 21.04, so it's not 'completed'.
I was able to secure the log file you asked before.
[config.log](https://github.com/edouard-lopez/record-gif.sh/files/10850405/config.log)

I still had to move `apt install` a bit earlier to fix it.

**Comment by edouard-lopez** [comment]
I pushed a fix on master with 1f230d2
